app/views/two_factor_authentication/otp_verification/show.html.erb

```diff
@@ -19,7 +19,8 @@
           value: @presenter.code_value,
           required: true,
           autofocus: true,
-          numeric: false,
+          numeric: IdentityConfig.store.enable_numeric_authentication_otp ? true : false,
```

Comment:
Is the ternary redundant if the configuration value is already a boolean?

```suggestion
          numeric: IdentityConfig.store.enable_numeric_authentication_otp,
```

Reply:
Yes 😀

Reply:
@zachmargolis smartly pointed out that we only want to enable numeric input once all instances are sending numeric.  

If we enable numeric input at the same time as enabling numeric OTPs, it's possible a user gets sent an alphanumeric OTP from the old version but ends up with a page rendered with the new version where the input is numeric.

To avoid that, I added a separate config flag `enable_numeric_authentication_otp_input` that we can enable once we have completely deployed with `enable_numeric_authentication_otp` enabled.